doc/user/backup_cronjob/README.md

```diff
@@ -0,0 +1,43 @@
+# Periodic etcd Backup using CronJob
+
+This doc talks about how to use Kubernetes [CronJob][k8s_cronjob] to make periodic etcd backups.
+
+Prerequisites:
+
+- Kubernetes cluster and kubectl ready
+- etcd-operator, etcd-backup-operator deployed
+- `example-etcd-cluster` EtcdCluster CR is created and the cluster is running
+
+All commands assume pwd in the root dir of etcd-operator repo.
+We are also following the example configurations in [backup-operator walkthrough][backup-operator-walkthrough].
+
+
+## Backup ConfigMap
+
+First we create a ConfigMap containing the Backup CR template for the CronJob container to mount and reuse it.
+
+Create the ConfigMap:
+
+```
+kubectl create -f ./doc/user/backup_cronjob/configmap.yaml
+```
+
+## Backup CronJob
+
+Then we create a CronJob which would create backup CR periodically.
+
+```
+kubectl create -f ./doc/user/backup_cronjob/cronjob.yaml
```

Comment:
we should talk about what this cron job does.

Reply:
what is the name format, what is the default duration? where to change the duration type of thing.